manticore/core/manticore.py

```diff
@@ -278,6 +283,69 @@ def _join_workers(self):
             while len(self._workers) > 0:
                 self._workers.pop().join()
 
+    #
+    # Symbol resolution
+    #
+
+    def _addr_by_symline(dwarfinfo, symbols, line):
+        '''
+        Return address for a symbol and line it is on through DWARF
+        debug info
+        '''
```

Comment:
Please use triple `"` characters for docstrings (yeah I know we have it mixed, we have to switch them all).